### PR TITLE
PBI #145159: Manual popup placement option

### PR DESCRIPTION
**Pull request by Kashkovsky**

### CODE REVIEW

File: src/components/Content/Popup/Popup.js [hunk 1]
```diff
@@ -93,7 +95,7 @@ Popup.propTypes = {
 		getBoundingClientRect: PropTypes.func
 	}),
 	/** One of the placements supported by Popper.js relative to target node. */
-	placement: PropTypes.string,
+	placement: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
```
Comment (dmarcautan) [inline src/components/Content/Popup/Popup.js]: Add comment about False value

Comment (dmarcautan) [inline src/components/Content/Popup/Popup.js]: Can PropTypes.bool be replaced with 'false' or PropTypes.exact(false)?

Comment (Kashkovsky) [reply src/components/Content/Popup/Popup.js]: Done

Comment (Kashkovsky) [reply src/components/Content/Popup/Popup.js]: react2dts tool cannot properly parse `false` and `PropTypes.exact(false)` resulting in `any` type

**Pull request merged**